views/sections/forms/contact.ejs

```diff
@@ -0,0 +1,57 @@
+<div class="container-sm align-items-center">
+    <div class="py-2 mt-4 align-items-center">
+        <h1>Bize Ulaşın</h1>
+        <!-- Registeration Form -->
+        <div class="ml-5 pt-3 align-items-center form-container">
+            <form action="#">
+                <div class="">
+                    <!-- First Name -->
+                    <div class="input-group col-lg-6 mb-4">
+                        <div class="input-group-prepend">
+                            <span class="px-1 border-md border-right-0 me-2">
+                                <img class="" src="/public/icon/profile.png">
+                            </span>
+                        </div>
+                        <input id="firstName" type="text" name="firstname" placeholder="İsim" class="form-control bg-white border-left-0 border-md">
+                    </div>
+
+                    <!-- Last Name -->
+                    <div class="input-group col-lg-6 mb-4">
+                        <div class="input-group-prepend">
+                            <span class="px-1 border-md border-right-0 me-2">
+                                <img class="" src="/public/icon/profile.png">
+                            </span>
+                        </div>
+                        <input id="lastName" type="text" name="lastname" placeholder="Soyisim" class="form-control bg-white border-left-0 border-md">
+                    </div>
+
+                    <!-- Email Address -->
+                    <div class="input-group col-lg-12 mb-4">
+                        <div class="input-group-prepend">
+                            <span class="px-1 border-md border-right-0 me-2">
+                                <img class="" src="/public/icon/mail.png">
+                            </span>
+                        </div>
+                        <input id="email" type="email" name="email" placeholder="Email Address" class="form-control bg-white border-left-0 border-md">
+                    </div>
+
+                    <div class="input-group col-lg-6 mb-4">
+                        <div class="input-group-prepend">
+                            <span class="px-1 border-md border-right-0 me-2">
+                                <img class="" src="/public/icon/message.png">
+                            </span>
+                        </div>
+                        <input id="message" type="text" name="firstname" placeholder="Mesajınız" class="form-control bg-white border-left-0 border-md">
+                    </div>
```

Comment:
You should use `text area` rather than just input field, because messages won't be short like other inputs (first name, last name, email). You can refer [here](https://getbootstrap.com/docs/4.0/components/forms/#form-controls) for an example text area.

Reply:
OK,will work on these and re-request review here.

Reply:
Please do not re-request something. You just continue to push your new commits into your branch. They will automatically appear here. When the proper changes have been made, I'll resolve the conversation(s).

Reply:
Sorry if mistaken, I meant to inform you about the changes I've made not opening another pull request.

<img width="430" alt="Screen Shot 2022-10-03 at 01 17 49" src="https://user-images.githubusercontent.com/40958091/193478798-722ef950-e7dd-490f-ba41-c3c5d84f5e66.png">